src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -8,15 +8,15 @@ build-backend = "poetry.core.masonry.api"
 build-system-version = "1.5.1"
 
 [tool.poetry]
-name = "{{cookiecutter.__pkg_name}}"
-version = "{{cookiecutter.__version}}"
-description = "{{cookiecutter.__short_description}}"
+name = "ansys-solutions-calculator"
```

Comment:
@iazehaf does it mean there cannot be more than one osl solution installed, because they will all have the same name?

Reply:
Given a owa archive, we can have only one osl solution generated from the owa in the current working directory. Nothing prevent you from generating multiple solutions from the same owa but you will need to change directory to avoid overriding the existing solution.